tests/integration/long/test_ssl.py

```diff
@@ -41,12 +49,21 @@ def setup_module():
     ccm_cluster.stop()
 
     # Fetch the absolute path to the keystore for ccm.
-    abs_path_server_keystore_path = os.path.abspath(DEFAULT_SERVER_KEYSTORE_PATH)
+    abs_path_server_keystore_path = os.path.abspath(SERVER_KEYSTORE_PATH)
 
     # Configure ccm to use ssl.
+
     config_options = {'client_encryption_options': {'enabled': True,
                                                     'keystore': abs_path_server_keystore_path,
                                                     'keystore_password': DEFAULT_PASSWORD}}
+
+    if(client_auth):
+        abs_path_server_truststore_path = os.path.abspath(SERVER_TRUSTSTORE_PATH)
+        client_encyrption_options = config_options['client_encryption_options']
+        client_encyrption_options['require_client_auth'] = True
+        client_encyrption_options['truststore'] = abs_path_server_truststore_path
+        client_encyrption_options['truststore_password'] = DEFAULT_PASSWORD
+
     ccm_cluster.set_configuration_options(config_options)
     ccm_cluster.start(wait_for_binary_proto=True, wait_other_notice=True)
```

Comment:
Something is causing this to fail when run as follows:

    nosetests -s -v tests/integration/long/test_ssl.py

ccm (2.0.3.1), java version "1.8.0_45"

I haven't quite figured this out.